src/components/Navigation/Header.tsx

```diff
@@ -55,8 +56,14 @@ const Header = ({ pathname, toggleMobileNav }: HeaderProps) => {
           </tds-header-dropdown-list-item>
         </tds-header-dropdown-list>
       </tds-header-dropdown>
-      <tds-header-item selected={pathname === "/text"}>
-        <Link to="text">Text</Link>
+      <tds-header-item
+        selected={pathname === "/lorem-ipsum/text-with-a-very-long-title"}
+      >
+        <Link to="lorem-ipsum/text-with-a-very-long-title">
+          <div className="header-item-text-wrap">
+            Text with a very long title
+          </div>
+        </Link>
```

Comment:
What if this one was linked to "/lorem-ipsum", and that page then had a link in it to take you to "lorem-ipsum/text-with-a-very-long-title", which is not visible in the header. Then the breadcrumb would provide a more lifelike scenario maybe?

Reply:
Good idea